packages/components/src/components/data-grid/cell-handlers/cell-interface.ts

```diff
@@ -17,6 +17,8 @@ export interface Cell {
     resizable?: boolean;
     sortable?: boolean;
     sortBy?: 'number' | 'text' | 'date';
+    presort?: boolean;
+    presortDirection?: 'ascending' | 'descending';
```

Comment:
or 'presortDirectionAscending: boolean' to have boolean, not strings? but seems shitty